### PR TITLE
Close dkm application

### PR DESCRIPTION
**Pull request by PushingCoder572**
## Describe your changes

Change the sidebar to not say DKM is currently not recruiting. (better late than never :))

## Checklist before requesting a review

- [x] I have performed a self-review of my changes
- [x] I have updated both Swedish and English pages (if not motivate why)

### CODE REVIEW

File: en/clubs/dkm/sidebar.md [hunk 1]
```diff
@@ -7,7 +7,8 @@ Complaints, compliments, gifts?</br>
 **DKM President (Klubbmästare):**  [klubbm@datasektionen.se](mailto:klubbm@datasektionen.se)
 
 Assortment and other bar-related stuff?</br>
-**DKM Bar responsible (Barmästare):** [barm@d.kth.se](mailto:barm@d.kth.se)
+**DKM Bar responsible (Barmästare):** [barm@d.kth.se](mailto:barm@datasektionen.se)
```
Comment (Herkarl) [inline en/clubs/dkm/sidebar.md:10]: Was about to comment that this email did not exist last I checked, but it seems to have been added now. Nice! 🎉

**Review by Herkarl — APPROVED** [review]
Lgtm, small spelling thing, but fix if you want

File: en/clubs/dkm/sidebar.md [hunk 2]
```diff
@@ -7,7 +7,8 @@ Complaints, compliments, gifts?</br>
...
 
 ## Apply to DKM
-Applications to DKM are open until the 12th of October! Apply at [dkm.io/ansok](www.dkm.io/ansok)!
+
+Applications to DKM are currently not open. We usualy recruiting during the fall.
```
Comment (Herkarl) [inline en/clubs/dkm/sidebar.md:14]:
```suggestion
Applications to DKM are currently not open. We are usually recruiting during the fall.
```

File: en/clubs/dkm/sidebar.md (hunk 1 above)
Comment (Herkarl) [inline en/clubs/dkm/sidebar.md:10]:
```suggestion
**DKM Bar responsible (Barmästare):** [barm@datasektionen.se](mailto:barm@datasektionen.se)
```

File: namnder/dkm/sidebar.md [hunk 3]
```diff
@@ -7,8 +7,8 @@ Klagomål, komplimanger, gåvor?</br>
 **Klubbmästare:**  [klubbm@datasektionen.se](mailto:klubbm@datasektionen.se)
 
 Sortiment och annat bar-relaterat?</br>
-**Barmästare:** [barm@datasektionen.se](mailto:barm@d.kth.se)
+**Barmästare:** [barm@datasektionen.se](mailto:barm@datasektionen.se)
 
 ## Ansökan till DKM
 
-Ansökan till DKM är öppen tills den 12e Oktober! Sök på DKM på [dkm.io/ansok](www.dkm.io/ansok)
+Ansökan till DKM är inte öppen för tillfället. Vi brukar rekturera under tidig höst.
```
Comment (elmaxe) [inline namnder/dkm/sidebar.md:14]:
```suggestion
Ansökan till DKM är inte öppen för tillfället. Vi brukar rekrytera under tidig höst.
```

**Review by elmaxe — DISMISSED** [review]

**Review by Herkarl — CHANGES_REQUESTED** [review]
Sorry for the many reviews, realised the displayed email was wrong in the english page

**Review by Herkarl — APPROVED** [review]
Nice! :+1: 
LGTM

**Review by elmaxe — APPROVED** [review]

**Pull request merged**